setup.cfg

```diff
@@ -1,24 +1,26 @@
-[bdist_wheel]
-universal=1
+[bumpversion]
+current_version = 3.0.0
+commit = True
+tag = True
 
-[metadata]
-license_file = LICENSE.txt
+[bumpversion:file:setup.py]
+search = version='{current_version}'
+replace = version='{new_version}'
 
-[aliases]
-test=pytest
+[bumpversion:file:quilt3distribute/__init__.py]
```

Comment:
`quilt3distribute`?

Reply:
Whoops, should have searched for all `quilt3distribute` references when I copied the build procedure stuff over...